sim/battle.ts

```diff
@@ -1455,7 +1455,7 @@ export class Battle extends Dex.ModdedDex {
 				return true;
 			}
 			const turnsLeftText = (turnsLeft === 1 ? `1 turn` : `${turnsLeft} turns`);
-			this.add('bigerror', `You will auto-tie if the battle doesn't end in ${turnsLeftText} turns (on turn 1000).`);
+			this.add('bigerror', `You will auto-tie if the battle doesn't end in ${turnsLeftText} turns (on turn 1000). Consider using '/offertie' to see if your opponent would be willing to accept a tie earlier if you feel this battle is a stalemate.`);
```

Comment:
Maybe not this entire message as a `bigerror`?

Reply:
Made this a `hint`. Does ` `` `   (I still cant escape it properly on GH, and I dont care) work the way I hope?

Reply:
Man, don't resolve a conversation immediately after asking a question.

Anyway, I don't actually know.

Reply:
I just tested and it turns out the answer is "no".